### DISCUSSION TITLE
Move the 'pop-out race control messages' button outside the livetiming

### DISCUSSION THREAD

**Discussion by JustJoostNL** [f1multiviewer/issue-tracker#72] [Feature requests]
Some people already asked for a separate race control page, this is awesome! But sometimes my pc doesnt like many feeds and live timing+racecontrol, in that case i only want the racecontrol messages, and not the livetiming. Now if you close the livetiming, the racecontrol messages also closes. If this can be moved outside the livetiming, or let us close the livetiming without the racecontrol messages closing. That would be great! 😀

**Comment by SimplyBlu** [comment]
Would be nice to take this idea plus #66 (stay on top plus borderless window) for a better race control message experience.
